test/FixedPoint.ts

```diff
@@ -1,6 +1,6 @@
 import { BigNumber } from 'ethers';
 
-const PRECISION = 1000000000000;
+const PRECISION = 100000000;
```

Comment:
This was a precision that is used when converting the display price into sqrt (Q64), having greater precision is better, but apparently it broke the unit test because of some bignumber overflow issue, so i reverted back to the original one.